lib/metasploit/framework/login_scanner/winrm.rb

```diff
@@ -39,13 +40,33 @@ def set_sane_defaults
           super
         end
 
+        def parse_auth_methods(resp)
+          return [] unless resp and resp.code == 401
+          methods = []
+          methods << "Negotiate" if resp.headers['WWW-Authenticate'].include? "Negotiate"
```

Comment:
@dwelch-r7 Under what circumstances will winrm return this header? I haven't found a target that sends this back yet.